claim-check/code-samples/sample-4/azure-function.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
This should be sample-4.sln like the previous samples.  Also, remove the solution files in the directories below this one.  This sample doesn't open from the root in VS Code like the others.

This sample doesn't appear to have been changed since the initial review.

Reply:
Done

Reply:
The solution file in the root has the wrong project filename for the Azure function, so it doesn't load properly.